docs/github_projects_guidelines.md

```diff
@@ -0,0 +1,478 @@
+# Github projects guidelines
+
+This document describes guidelines and recomendations that should be used when working with anyone of the project hosted on GitHub.
+
+## Table of Contents 📜
```

Comment:
I don't think this is needed here, since github offers navigation by clicking on the hamburger menu button on the top.
Less scrolling needed...

Reply:
I find it nicer/practical to have a ToC always visible.
Reasoning: whenever you will open this document you will want to jump to a specific section. So it is most practical to click on a ToC link (whenever visible as it is now or behind a hamburger icon), as you will not want to scroll.
So you will want to use ToC links all the time, that's I think that they should be offered explicitly and not implicitly behind a hamburger icon.